smart_aquarium/lib/ui/SensorsPage.dart

```diff
@@ -27,8 +27,13 @@ class _SensorsPageState extends State<SensorsPage> {
   Future<List<dynamic>> _fetchSensors() async {
     SharedPreferences prefs = await SharedPreferences.getInstance();
     int aquariumId = prefs.getInt('aquarium_id') ?? 0;
+    String token = prefs.getString('token') ?? '';
 
-    final response = await http.get(Uri.parse('$baseUrl/api/v1/userSensor'));
+    final response =
+        await http.get(Uri.parse('$baseUrl/api/v1/userSensor'), headers: {
+      'Content-Type': 'application/json',
+      'Authorization': 'Bearer $token',
+    });
     if (response.statusCode == 200) {
```

Comment:
Maybe you could add some global constants, in the same place where baseUrl const is, for error codes, because now they are some magic numbers currentyl. It will make reading code easier. You should add constants only for codes used in the program and not for all available http codes
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status